### ISSUE TITLE
[Sourcing-2][Accessibility: ACC-270.6] List Item: UI cannot be activated with keyboard.

### ISSUE THREAD

**Issue by divyamalhotra488** [SAP/fundamental-ngx#11963]
#### Is this a bug, enhancement, or feature request?
Bug

#### Describe your proposal.
Component having issue: https://sap.github.io/fundamental-ngx/#/core/list
The items within the list, cannot be activated using keyboard.
It must be possible to invoke/activate each & every interactive UI, using the keyboard.

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
All latest versions.

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
Use any list item. check on select of items within the list, isn't activated using keyboard.

#### Please provide relevant source code (if applicable).
Provided stackblitz example below.

#### Please provide stackblitz example(s).
https://stackblitz.com/edit/yvrbeb?file=src%2Fapp%2Flist-selection-example.component.html

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations.
Yes we have got this bug from Accessibility expert only.

#### Did you check the documentation and the API?
Yes

#### Is there anything else we should know?
Re-raising the issue as it was closed earlier without fix (please refer https://github.com/SAP/fundamental-ngx/issues/11292)

**_IMPORTANT: Please refrain from providing links or screenshots of SAP's internal information, as this project is open-source, and its contents are accessible to anyone._**

**Comment by mikerodonnell89** [comment]
This video, recording from your stackblitz, is using only tab/arrow/enter/space keypresses (you can see the mouse cursor on the title text):

https://github.com/user-attachments/assets/bb0f414c-1bb4-49a6-9010-6ccd1d0c3956
